### PR TITLE
Bump Bio-Formats version to 6.10.0

### PR DESCRIPTION
**Pull request by dgault**
Bumping BF version to 6.10.0

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <properties>
     <ome-common.version>6.0.6</ome-common.version>
```
Comment (sbesson) [inline pom.xml:26]:
```suggestion
    <ome-common.version>6.0.9</ome-common.version>
```

File: pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
...
     <ome-model.version>6.2.2</ome-model.version>
```
Comment (sbesson) [inline pom.xml:27]:
```suggestion
    <ome-model.version>6.3.1</ome-model.version>
```

**Review by sbesson — COMMENTED** [review]
Two suggestions to update the version of the low-level components as well since they are being bumped. Otherwise, reads well to me

File: pom.xml (hunk 2 above)
Comment (dgault) [reply pom.xml:27]: Sorry, missed this comment, follow PR open now: https://github.com/ome/bio-formats-documentation/pull/256

**Pull request merged**